src/mapogcapi.cpp

```diff
@@ -301,6 +301,11 @@ static bool getBbox(mapObj *map, layerObj *layer, cgiRequestObj *request,
     bbox->maxx = values[2];
     bbox->maxy = values[3];
 
+    if (bbox->minx > bbox->maxx) {
+      // the bbox can span the antimeridian and give the
+      // lower-value first - in this case swap the values
+      std::swap(bbox->minx, bbox->maxx);
```

Comment:
Unfortunately the fix would be much more complicated than just switching minx and maxx.

We should actually issue 2 queries:
- one with [bbox->minx, 180] range
- another one with [-180, bbox->maxx] range
and merge the results

Reply:
Ah of course, this is going to get the completely wrong results.. I presume this issue wasn't addressed in WFS queries? Will look now.

Reply:
> I presume this issue wasn't addressed in WFS queries?

no, WFS hasn't any such special processing for antimeridian crossing